pages/api/labs/games/multiplication-connect/Index.test.ts

```diff
@@ -0,0 +1,169 @@
+import { calculateWinner, createGrid, SelectedBy } from "../../../../studentPortal/labs/multiplication-connect/Index";
+
+// Arrange
+// Act
+// Assert
+
+/* To set the test up:
```

Comment:
Do you want to keep all the comments in here?

Reply:
No I don't, thanks for reminding me!